### PR TITLE
[4.0] Fix RTL for Sample Data Module

### PR DESCRIPTION
**Pull request by pritam825**
Pull Request for Issue #35850 (partly).

### Testing Instructions
first choose the arabic language then
go to home dashboard -> Sample Data

### Actual result BEFORE applying this Pull Request

Screenshot (1919)

### Expected result AFTER applying this Pull Request
Screenshot (1920)

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 646ba4cb3f76e90b498f11a9e6055054db4f25b4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36784">issues.joomla.org/tracker/joomla-cms/36784</a>.</sub>

**Comment by brianteeman** [comment]
Could you change the title of this PR to the name of the module please<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36784">issues.joomla.org/tracker/joomla-cms/36784</a>.</sub>

**Comment by Shubhamverma2796** [comment]
I have tested this item :white_check_mark: successfully on 646ba4cb3f76e90b498f11a9e6055054db4f25b4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36784">issues.joomla.org/tracker/joomla-cms/36784</a>.</sub>

**Comment by richard67** [comment]
@pritam825 I've allowed myself to adjust the title of your pull request as recommended by @brianteeman in his [comment above](https://github.com/joomla/joomla-cms/pull/36784#issuecomment-1019261623).

Please always chose meaningful and understandable titles for pull requests. Thanks in advance.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36784">issues.joomla.org/tracker/joomla-cms/36784</a>.</sub>

**Comment by pritam825** [comment]
@brianteeman sir, @richard67 sir updated the title.
@richard67 sir, Thanks for updating, Sorry sir for the wrong title, actually I was confused with the title, now understood, Thanks a lot for helping me @brianteeman sir and @richard67 sir.

File: administrator/modules/mod_sampledata/tmpl/default.php [hunk 1]
```diff
@@ -32,7 +32,7 @@
 		<?php foreach($items as $i => $item) : ?>
 			<li class="list-group-item sampledata-<?php echo $item->name; ?>">
 				<div class="d-flex justify-content-between align-items-center">
-					<div class="sample-data__title me-2">
+					<div class="sample-data__title ">
```
Comment (Quy) [inline administrator/modules/mod_sampledata/tmpl/default.php]:
```suggestion
					<div class="sample-data__title”>
```

File: administrator/modules/mod_sampledata/tmpl/default.php [hunk 2]
```diff
@@ -32,7 +32,7 @@
 		<?php foreach($items as $i => $item) : ?>
 			<li class="list-group-item sampledata-<?php echo $item->name; ?>">
 				<div class="d-flex justify-content-between align-items-center">
-					<div class="sample-data__title me-2">
+					<div class="sample-data__title”>
```
Comment (richard67) [inline administrator/modules/mod_sampledata/tmpl/default.php]:
```suggestion
					<div class="sample-data__title">
```
You should be careful to use the right kind of double quotes.

Comment (pritam825) [reply administrator/modules/mod_sampledata/tmpl/default.php]: @richard67 thanks, it was a nice catch

Comment (richard67) [reply administrator/modules/mod_sampledata/tmpl/default.php]: I only make nice catches 😛

**Comment by pritam825** [comment]
@richard67 , can you please have a look on this #36767 PR also, it would be great help, Thanks

**Comment by richard67** [comment]
I've restored the previous human test results since the changes after that were just removal of a redundant space. So still RTC (ready to commit).

**Comment by pritam825** [comment]
> I've restored the previous human test results since the changes after that were just removal of a redundant space. So still RTC (ready to commit).

@richard67 thanks, I have opened an another PR also, which haven't confirmed yet that, it's right or wrong, can you please please confirm this PR #36767 also, Thanks in advance :)

**Comment by richard67** [comment]
> @richard67 , can you please have a look on this #36767 PR also, it would be great help, Thanks

> @richard67 thanks, I have opened an another PR also, which haven't confirmed yet that, it's right or wrong, can you please please confirm this PR #36767 also, Thanks in advance :)

It does not need me to tell that 2 times.

And I am able to decide myself which PRs I am looking on.

**Comment by pritam825** [comment]
Ok sir, Thanks :)

**Comment by bembelimen** [comment]
Thx

**Comment by pritam825** [comment]
Thanks for merging

**Pull request merged**